log4j-plugins/src/main/java/org/apache/logging/log4j/plugins/util/PluginRegistry.java

```diff
@@ -320,10 +326,12 @@ public void loadPlugins(ClassLoader classLoader, Map<String, List<PluginType<?>>
         }
 
         final long endTime = System.nanoTime();
```

Comment:
Can we move these nanoTime invocations into the conditional as well?

Reply:
This is a good idea but it makes more sense to me to have the whole message generated as a lambda. Working on modifying it to do that now.